bindata/network/additional-networks/sriov/sriov-cni.yaml

```diff
@@ -0,0 +1,42 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: sriov-cni
+  namespace: openshift-sriov
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the SR-IOV CNI plugin on each node.
```

Comment:
"This daemonset copies the SR-IOV CNI plugin on to each node"

Reply:
Done